host/sgx/linux/sgxquoteexloader.c

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <dlfcn.h>
+#include <openenclave/internal/defs.h>
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/trace.h>
+#include <stdlib.h>
+#include <string.h>
+#include "../../../common/oe_host_stdlib.h"
+#include "../sgxquote_ex.h"
+
+static const char* _quote_ex_library_file_name = "libsgx_quote_ex.so.1";
+
+static oe_sgx_quote_ex_library_t* _quote_ex_library = NULL;
+
+static void _unload_quote_ex_library(void)
+{
+    if (_quote_ex_library && _quote_ex_library->handle)
+    {
+        OE_TRACE_INFO(
+            "_unload_quote_ex_library() %s\n", _quote_ex_library_file_name);
+        dlclose(_quote_ex_library->handle);
+        _quote_ex_library->handle = 0;
+        if (_quote_ex_library->mapped)
+        {
+            oe_free(_quote_ex_library->mapped);
+            _quote_ex_library->mapped = NULL;
+        }
+        if (_quote_ex_library->uuid)
+        {
+            oe_free(_quote_ex_library->uuid);
+            _quote_ex_library->uuid = NULL;
+        }
+        if (_quote_ex_library->sgx_key_id)
+        {
+            oe_free(_quote_ex_library->sgx_key_id);
+            _quote_ex_library->sgx_key_id = NULL;
+        }
+    }
+}
+
+void oe_sgx_load_quote_ex_library(oe_sgx_quote_ex_library_t* library)
+{
+    oe_result_t result = OE_UNEXPECTED;
+
+    if (!library)
+        OE_RAISE(OE_INVALID_PARAMETER);
+
+    _quote_ex_library = library;
+
+    if (library->handle == 0)
+    {
+        void* _handle = 0;
```

Comment:
As noted in my previous comment, this variable is not static (it's "void*" not "static void*") and so should not start with _ per OE coding style guidelines.

Reply:
Oh got it. I mistook your reference as to `_quote_ex_library`. The variable name `_handle` was borrowed from existing file `host/sgx/windows/sgxquoteproviderloader.c`. Will fix the naming in both the Linux and Windows versions.